src/styles/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  transition: transform 0.3s ease;
+  color: $black;
+
+  &:hover {
+    transform: scale(120%);
+    color: #34568b;
+  }
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: 160px 134px;
+    height: 166px;
+    background-position: 19px 32px;
+  }
+
+  &__apple {
+    margin: $margin;
+    padding: 40px 16px 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__product {
+    margin: $margin;
+    padding: 4px 16px 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: $grey;
+  }
+
+  &__price {
+    width: $width;
+    height: 18px;
+    display: flex;
+    padding: 24px 16px 16px;
+    justify-content: space-between;
+
+    &_text {
+      font-size: 12px;
+      margin: 0;
+      line-height: 18px;
+      color: $grey;
+    }
+
+    &_amount {
+      font-size: 16px;
+      font-weight: 700;
+      margin: 0;
+      line-height: 18px;
+      color: $black;
+    }
+  }
+
+  &__box {
+    padding: 0 16px 16px;
+  }
+
+  &__button {
+    width: $width;
```

Comment:
The variable `$width` is used here without being defined. Ensure it is defined elsewhere or replace it with a specific value.